app/elastic_search_indexes/search/goods_nomenclature_index.rb

```diff
@@ -83,6 +83,8 @@ def definition
             goods_nomenclature_class: { type: 'keyword' },
             goods_nomenclature_item_id: { type: 'text' },
             producline_suffix: { type: 'keyword' },
+            chapter_id: { type: 'keyword' },
```

Comment:
Since its a requirement to filter these now we need to make sure they are backed by keyword indexes